functions/exercises.py

```diff
@@ -113,12 +127,14 @@ def check_num_in_range(number, range_min, range_max):
     :return:
     """
     # INSERT YOUR CODE HERE
-    #
+    if number > range_min and number < range_max:
+        return True
+    else:
+        return False

```

Comment:
```suggestion
    return number > range_min and number < range_max:
```
Do we need this ifs?